db/init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE IF NOT EXISTS rha;
+
+CREATE TABLE events (
+    event_id BIGINT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
```suggestion
    id BIGINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
    event_id VARCHAR(12) DEFAULT NULL,
```

let's also keep a non-numeric, non-incremental public facing id in the DB along with the numeric `id`. Numeric ids make it easy to iterate over or guess other users / events. Making non-numeric ids primary key will lead to performance issues. This is a middle ground between both.

- https://planetscale.com/blog/why-we-chose-nanoids-for-planetscales-api
- https://planetscale.com/blog/the-problem-with-using-a-uuid-primary-key-in-mysql